src/cmds/pbs_ralter.c

```diff
@@ -51,6 +51,7 @@
 static struct attrl *attrib = NULL;
 static time_t dtstart;
 static time_t dtend;
+int force_alter = FALSE;
```

Comment:
I can make it a static, but I do want to initialize explicitly because if TRUE or FALSE ever change their value (highly unlikely) then it may not work.
If you feel strongly about this change then let me know, I'll make it.

Reply:
Initializing it is fine.  I personally don't think any of those need to be static since this is a command, and we don't link them into other programs.  I was just thinking it should be consistent with what is there.